src/db.rs

```diff
@@ -1649,6 +1649,47 @@ impl WriteBatch {
         }
     }
 
+    /// Remove database entries from start key to end key.
+    ///
+    /// Removes the database entries in the range ["begin_key", "end_key"), i.e.,
+    /// including "begin_key" and excluding "end_key". It is not an error if no
+    /// keys exist in the range ["begin_key", "end_key").
+    pub fn delete_range<K: AsRef<[u8]>>(&mut self, from: K, to: K) -> Result<(), Error> {
+        let (start_key, end_key) = (from.as_ref(), to.as_ref());
+
+        unsafe {
+            ffi::rocksdb_writebatch_delete_range(
+                self.inner,
+                start_key.as_ptr() as *const c_char,
+                start_key.len() as size_t,
+                end_key.as_ptr() as *const c_char,
+                end_key.len() as size_t,
+            );
+            Ok(())
+        }
+    }
+
+    pub fn delete_range_cf<K: AsRef<[u8]>>(
```

Comment:
Sorry if I'm too carping but I'd like to see some doc-comments for this PUBLIC method. It's because the `docs.rs` service won't be able to create a description for the public method. And I suppose it's bad. Any thoughts ?

Reply:
Agreed. I think it's the last thing we need before a merge.

Reply:
Exactly !

Reply:
Sure, I'm on it!

Reply:
All done.